posts/2023-06-29-OutreachyInternshipExperience/index.qmd

```diff
@@ -0,0 +1,235 @@
+---
+title: "Our Journey as Outreachy Interns with Bioconductor"
+description: |
+  Bioconductor participated in the Outreachy Internship program for the May 2023 cohort. Join us as we share our enriching experience working on various projects at Bioconductor and delve into the world of open science.
+author:
+  - name: Sonali Kumari
+    url: https://github.com/sonali8434
+  - name: Daena Rys
+    url: https://github.com/Daenarys8
+  - name: Atrayee Samanta
+    url: https://github.com/atrayees
+date: 2023-06-29
+categories:
+  - open science
+---
+
+## Introduction
+
+The world of open source offers abundant opportunities for growth and impact.
+However, diving into it can be daunting for newcomers. Thankfully, initiatives
+exist to bridge the gap between aspiring contributors and open source
+communities.
+
+Outreachy is one such project providing internships to underrepresented
```

Comment:
I just want clarify the relationship between Outreachy and Bioconductor a little more with

`Outreachy is one such project that partners with open source and open science communities to provide internships to underrepresented`